website/views/pages/homepage.ejs

```diff
@@ -12,7 +12,15 @@
             <h6 style="max-width: 580px;" class="mx-auto mt-3 mb-0">
               Quickly deploy <a style="color: white; text-decoration-line: underline;" href="https://osquery.io/">osquery</a> and scale your fleet to 100,000+ devices on top of a stable core technology.
             </h6>
-            <a style="max-width: 280px" class="btn btn-block btn-primary px-3 px-sm-5 py-3 mx-auto my-4" href="/get-started">Get started with Fleet</a>
+            <div style="max-width: 480px;" class="mt-5 mx-auto mb-0 d-sm-flex">
+              <a style="max-width: 280px" class="d-sm-flex align-items-center justify-content-center btn btn-block btn-primary px-3 px-sm-5 py-3 mx-auto" href="/get-started">Get started with Fleet</a>
+              <a target="_blank" style="font-size: 20px; color: white; text-decoration: none; font-weight: 700;" class="d-sm-flex align-items-center justify-content-center btn px-3 py-3" href="https://www.youtube.com/watch?v=dg1dwGQMaLw">
```

Comment:
```suggestion
              <a target="_blank" style="font-size: 20px; color: white; text-decoration: none; font-weight: 700;" class="d-sm-flex align-items-center justify-content-center btn btn-link px-3 py-3" href="https://www.youtube.com/watch?v=dg1dwGQMaLw">
```

Reply:
@mike-j-thomas, I'd suggest adding this additional bootstrap class to get rid of the border that shows up after you click the video button.

<img width="640" alt="Screen Shot 2021-07-14 at 9 05 25 AM" src="https://user-images.githubusercontent.com/73313222/125636333-03ebb212-0703-4ff5-8c8f-3adea5b2dd67.png">